copier/main.py

```diff
@@ -202,6 +202,8 @@ class Worker:
     context_lines: PositiveInt = 3
     unsafe: bool = False
 
+    answers: AnswersMap = field(default_factory=AnswersMap, init=False)
```

Comment:
I'd rename it now, to avoid pydantic making it a parameter: 

```suggestion
    _answers: AnswersMap = field(default_factory=AnswersMap, init=False)
```

Reply:
It's not a Pydantic parameter because Pydantic respects `field(init=False)`:

```python
from pydantic.dataclasses import dataclass
from dataclasses import field


@dataclass
class A:
    x: int = field(default=1, init=False)


A(x=2)
```

raises an error:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File ".venv/lib/python3.9/site-packages/pydantic/dataclasses.py", line 322, in new_init
    handle_extra_init(self, *args, **kwargs)
  File ".venv/lib/python3.9/site-packages/pydantic/dataclasses.py", line 286, in handle_extra_init
    init(self, *args, **{k: v for k, v in kwargs.items() if k in self.__dataclass_fields__})
TypeError: __init__() got an unexpected keyword argument 'x'
```

Reply:
Ah! I overlooked that. Thanks

Reply:
No problem 🙂